driver/node.go

```diff
@@ -164,6 +166,13 @@ func (s *NodeService) NodeGetCapabilities(ctx context.Context, req *proto.NodeGe
 					},
 				},
 			},
+			{
+				Type: &proto.NodeServiceCapability_Rpc{
```

Comment:
Adding this capability adds additional gRPC calls made for volumes in other Container Orchestrators. I would prefer to wait for moby/swarmkit#3116 as Swarm is not csi-compliant right now.

Reply:
Hmm, as an alternative, would you be fine with hiding this behind a feature flag that we can enable/disable via an ENV var that we automatically set in the Docker plugin?

I totally understand your point though and once the PR is merged, I would be happy to provide another PR that removes the flag.